include/seqan3/range/view/to_lower.hpp

```diff
@@ -0,0 +1,69 @@
+ * \returns             A range of converted elements. See below for the properties of the returned range.
+ * \ingroup view
+ *
+ * ### View properties
+ *
+ * This view is a **deep view:** Given a range-of-range as input (as opposed to just a range), it will apply
+ * the transformation on the innermost range (instead of the outermost range).
+ *
+ * | range concepts and reference_t  | `urng_t` (underlying range type)      | `rrng_t` (returned range type)                     |
+ * |---------------------------------|:-------------------------------------:|:--------------------------------------------------:|
+ * | std::ranges::InputRange         | *required*                            | *preserved*                                        |
+ * | std::ranges::ForwardRange       |                                       | *preserved*                                        |
+ * | std::ranges::BidirectionalRange |                                       | *preserved*                                        |
+ * | std::ranges::RandomAccessRange  |                                       | *preserved*                                        |
+ * | std::ranges::ContiguousRange    |                                       | *lost*                                             |
+ * |                                 |                                       |                                                    |
+ * | std::ranges::ViewableRange      | *required*                            | *guaranteed*                                       |
+ * | std::ranges::View               |                                       | *guaranteed*                                       |
+ * | std::ranges::SizedRange         |                                       | *preserved*                                        |
+ * | std::ranges::CommonRange        |                                       | *preserved*                                        |
+ * | std::ranges::OutputRange        |                                       | *lost*                                             |
+ * | seqan3::const_iterable_concept  |                                       | *preserved*                                        |
+ * |                                 |                                       |                                                    |
+ * | seqan3::reference_t             | seqan3::char_concept                  | *preserved*                                        |
+ *
+ * See the \link view view submodule documentation \endlink for detailed descriptions of the view properties.
+ *
+ * ### Example
+ * \snippet test/snippet/range/view/to_case.cpp to_lower
+ * \hideinitializer
+ */
+inline auto const to_lower = deep{std::view::transform([] (auto const in) noexcept
+{
+	static_assert(char_concept<remove_cvref_t<decltype(in)>>, "The value type of seqan3::view::to_lower must model the seqan3::char_concept.");
```

Comment:
```suggestion
	static_assert(char_concept<remove_cvref_t<decltype(in)>>, 
	              "The value type of the innermost range must model the seqan3::char_concept.");
```
(line break if it is longer than 120 chars)